### DISCUSSION TITLE
Select cannot search when using getOptionLabelFromRecordUsing()

### DISCUSSION THREAD

**Discussion by TheFrankman** [filamentphp/filament#9510] [Help]
When i use ->getOptionLabelFromRecordingUsing() the search stops working. If i remove this, the search works fine, has anyone experiences this or have an insight into the cause : 

```
 Forms\Components\Select::make('artwork_id')
                            ->label('Select Existing Artwork')
                            ->relationship(name: 'artwork', titleAttribute: 'name', modifyQueryUsing: fn (Builder $query) => $query->where('organisation_id', $organisationId))
                            ->searchable(['name', 'original_name'])
                            ->preload()
                            ->getOptionLabelFromRecordUsing(function (Model $record) {
                                $path = $record->path;
                                $name = $record->name;
                              return  "<img src=\"$path\" alt=\"$name\" class=\"inline w-6 h-6 mr-2\" /> $name";
                            })
                            ->allowHtml(),
 ```
 
 When attempting to search it doesn't error, just shows no results.

**Comment by leandrocfe** [comment]
It was supposed to work. Could you share a repo on Github?

**Reply by TheFrankman** [reply]
I'm afraid it's a private repo so unfortunately not.
Might be able to knock something together quickly tomorrow that i can share

**Reply by TheFrankman** [reply]
This is going to sound like a lie, but it randomly started working. I tested it every which way. Booted back up the next morning and it worked. My bad.

**Reply by leandrocfe** [reply]
Ok, this was weird..

**Comment by lyrihkaesa** [comment]
Still cant search if getOptionLabelFromRecordUsing() with html?
